core/objects.go

```diff
@@ -603,3 +607,91 @@ type OCSPSigningRequest struct {
 	Reason    int
 	RevokedAt time.Time
 }
+
+type SignedCertificateTimestamp struct {
+	// The version of the protocol to which the SCT conforms
+	SCTVersion uint8 `db:"sctVersion",json:"rpcSCTVersion"`
+	// the SHA-256 hash of the log's public key, calculated over
+	// the DER encoding of the key represented as SubjectPublicKeyInfo.
+	LogID []byte `db:"logID",json:"rpcLogID"`
+	// Timestamp (in ms since unix epoc) at which the SCT was issued
+	Timestamp uint64 `db:"timestamp",json:"rpcTimestamp"`
+	// For future extensions to the protocol
```

Comment:
This will be required to assemble the SCT we want to put in OCSP responses in the future (but yes not now).